tests/integration-tests/configs/common/common.yaml

```diff
@@ -88,6 +88,31 @@ create:
         instances: {{ common.INSTANCES_DEFAULT_X86 }}
         oss: ["alinux2"]
         schedulers: ["slurm"]
+
+createami:
+  test_createami.py::test_build_image:
+    dimensions:
+      - regions: ["eu-west-3"]
+        instances: {{ common.INSTANCES_DEFAULT_X86 }}
+        oss: ["alinux2", "ubuntu2004", "centos7"]
+      - regions: ["us-gov-east-1"]
+        instances: {{ common.INSTANCES_DEFAULT_X86 }}
+        oss: ["ubuntu1804"]
+  test_createami.py::test_build_image_custom_components:
+    dimensions:
+      - regions: ["ap-southeast-2"]
+        instances: {{ common.INSTANCES_DEFAULT_X86 }}
+        oss: ["centos7", "ubuntu1804"]
+      - regions: ["us-east-1"]
```

Comment:
can we leave `["eu-west-1"]` as it was before?

Reply:
done